### DISCUSSION TITLE
exclude content from my send messages

### DISCUSSION THREAD

**Discussion by ProfessionalHorder** [DIGITALCRIMINALS/OnlyFans#629] [Ideas]
Hi, Now when I scrape messeges all pics and videos there downloaded. But I don't want the pics that I've uploaded. Could downloading your own uploads be a setting (and one that will be 'false' by default)?

**Comment by basketballhead6** [comment]
Second this. It would be a useful feature if content you have uploded in DMs is not downloaded if that is possible.

**Reply by 4nx13ty** [reply]
Is this still working? getting access denied errors when trying to use it.

**Reply by basketballhead6** [reply]
It worked for me earlier today. Are you sure you've got things set up correctly?
